src/main/java/org/elasticsearch/index/merge/scheduler/ConcurrentMergeSchedulerProvider.java

```diff
@@ -172,7 +172,9 @@ public void onRefreshSettings(Settings settings) {
                 logger.info("updating [{}] from [{}] to [{}]", MAX_MERGE_COUNT_KEY, ConcurrentMergeSchedulerProvider.this.maxMergeCount, maxMergeCount);
```

Comment:
Woops, sorry, you're right!  CMSProvider.this.maxMergeCount is the true value ...